src/Uno.UI/UI/Xaml/UIElement.macOS.cs

```diff
@@ -262,18 +262,16 @@ private protected override void OnNativeKeyUp(NSEvent evt)
 			{
 				if (!ClippingIsSetByCornerRadius)
 				{
-					if (Layer != null)
-					{
-						this.Layer.Mask = null;
-					}
+					Layer.Mask = null;
```

Comment:
As far as I remember, Layer could be null if not asked for, which is very different from iOS. Has that changed?

Reply:
no, you're 💯% right 😄. The mention

> This property is never nil. 
https://developer.apple.com/documentation/uikit/uiview/1622436-layer?language=objc

only applies to iOS, where it's `readonly`, and not macOS where something else could set if to `nil` at anytime. I'll fix the PR asap